SwiftEvolve/Tests/SwiftEvolveTests/RegressionTests.swift

```diff
@@ -10,7 +9,7 @@ class RegressionTests: XCTestCase {
     // Checks that we don't mess up the order of declarations we're not trying
     // to shuffle. In particular, if we store the properties in a Set or other
     // unordered collection, we could screw this up.
-    try SwiftLang.withParsedCode(
+    let code = try SyntaxParser.parse(source:
```

Comment:
Switching to `SyntaxParser.parse(source:)` instead of using a temp file is fine.